tools/PushToBlobFeed.targets

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <ItemsToPushGlobPattern Include="$(OutputPath)Packages\Microsoft.NET.Sdk.*.nupkg" />
-    <ItemsToPushGlobPattern Include="$(OutputPath)Packages\Microsoft.NET.Build.Extensions.*.nupkg" />
+    <ItemsToPushGlobPattern Include="$(RepositoryRootDirectory)bin\$(Configuration)\Packages\Microsoft.NET.Sdk.*.nupkg" />
```

Comment:
Why is it outputpath does not work here? This is duplicating this information, which is not ideal.